packages/atomic/src/components/common/tab-manager/tab-button.tsx

```diff
@@ -9,18 +9,23 @@ export interface TabButtonProps {
 
 export const TabButton: FunctionalComponent<TabButtonProps> = (props) => {
   const activeTabClass = props.isActive
-    ? "relative after:content-[''] after:block after:w-full after:h-1 after:absolute after:-bottom-0.5 after:bg-primary after:rounded"
+    ? "active-tab relative after:content-[''] after:block after:w-full after:h-1 after:absolute after:-bottom-0.5 after:bg-primary after:rounded"
```

Comment:
adding an empty class for tests is usually not the best solution. Maybe you could check the aria-current property instead !

Reply:
I don't think there's a locator for that... I'll look for a better solution!

Reply:
Actually I think can just do `.locator('[aria-current="true"]')`